src/components/forms/purchase-order-form.js

```diff
@@ -351,18 +356,23 @@ class PurchaseOrderForm extends React.Component {
                                     }}
                                 />
                             </div>
-                            <div className="col-md-2">
+                            <div className="col-md-4">
                                 <label> {T.translate("edit_purchase_order.promo_code")}</label>
-                                <Input
+                                <PromoCodeInput
```

Comment:
use this one https://github.com/OpenStackweb/openstack-uicore-foundation/blob/5cb567f9a4f402bafcfdef7ded2503457a35dc51/src/components/inputs/promocode-input.js#L18

Reply:
done